contracts/prices/derivatives/ChaiPriceSource.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.6.8;
+
+import "./IDerivativePriceSource.sol";
+
+/// @title Chai Price Source
+/// @author Melon Council DAO <security@meloncoucil.io>
+/// @notice Price source oracle for Chai
+contract ChaiPriceSource is IDerivativePriceSource {
+    address immutable public CHAI;
+    address immutable public DAI;
+    address immutable public DSR_POT;
+
+    constructor(address _chai, address _dai, address _dsrPot) public {
+        CHAI = _chai;
+        DAI = _dai;
+        DSR_POT = _dsrPot;
+    }
+
+    /// @notice Gets the rates for 1 unit of the derivative to its underlying assets
+    /// @param _derivative The derivative for which to get the rates
+    /// @return underlyings_ The underlying assets for the _derivative
+    /// @return rates_ The rates for the _derivative to the _underlyings
+    function getRatesToUnderlyings(address _derivative)
+        external
+        override
+        returns (address[] memory underlyings_, uint256[] memory rates_)
+    {
+        require(_derivative == CHAI, "getRatesToUnderlyings: only Chai is supported");
+
+        underlyings_ = new address[](1);
+        underlyings_[0] = DAI;
+        rates_ = new uint256[](1);
+        rates_[0] = __calcChaiRate();
+    }
+
+    /// @dev Calculation based on Chai source: https://github.com/dapphub/chai/blob/master/src/chai.sol
+    function __calcChaiRate() private returns (uint256) {
+        IPot pot = IPot(DSR_POT);
+        uint256 chi = (now > pot.rho()) ? pot.drip() : pot.chi();
+        return chi / 10 ** 9; // Refactor of mul(chi, 10 ** 18) / 10 ** 27
+    }
+}
+
+/// @notice Limited interface for Maker DSR's Pot contract
+/// @dev See DSR integration guide: https://github.com/makerdao/developerguides/blob/master/dai/dsr-integration-guide/dsr-integration-guide-01.md
+interface IPot {
+    function chi() external returns (uint256);
+    function rho() external returns (uint256);
```

Comment:
I'm just realizing that `chi()` and `rho()` are both public vars with getters built in, so can you please add `view` to these in the interface?